gravitee-apim-console-webui/src/management/instances/instance-list/instance-list.module.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { MatTableModule } from '@angular/material/table';
+import { MatIconModule } from '@angular/material/icon';
+import { MatTooltipModule } from '@angular/material/tooltip';
+import { UIRouterModule } from '@uirouter/angular';
+
+import { InstanceListComponent } from './/instance-list.component';
+
+import { GioTableWrapperModule } from '../../../shared/components/gio-table-wrapper/gio-table-wrapper.module';
+
+@NgModule({
+  declarations: [InstanceListComponent],
+  exports: [InstanceListComponent],
+  imports: [CommonModule, MatTableModule, GioTableWrapperModule, MatIconModule, MatTooltipModule, UIRouterModule],
+})
```

Comment:
We are trying to keep it clean, so by the order, MatModules all next to each other :)